src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -107,8 +156,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             reinterpret_cast<::IUnknown**>(factory.put())));
 
         // get the font collection; subscribe to updates
-        com_ptr<IDWriteFontCollection> fontCollection;
-        THROW_IF_FAILED(factory->GetSystemFontCollection(fontCollection.put(), TRUE));
+        const auto fontCollection = NearbyCollection(factory.get());
```

Comment:
So this will recreate it every time the page reloads the view model? Not a perf issue?

Reply:
From what I could tell this only loads once when the settings page is opened. Navigating back and forth between views didn't call this function again.